addons/api/mirage/factories/target.js

```diff
@@ -22,25 +22,23 @@ export default factory.extend({
       'add-credential-sources',
       'remove-credential-sources',
     ],
-
-  id: () => generateId('t_'),
+  id() {
+    return this.type === TYPE_TARGET_TCP
+      ? generateId('ttcp_')
+      : generateId('tssh_');
+  },
 
   /**
    * -1 means "unlimited" and we want to generate these on occasion.
    */
   session_connection_limit: () =>
     faker.helpers.arrayElement([-1, faker.datatype.number()]),
   worker_filter: (i) => (i % 2 === 0 ? faker.random.words() : null),
+  egress_worker_filter: (i) => (i % 2 !== 0 ? faker.random.words() : null),
+  ingress_worker_filter: (i) => (i % 2 !== 0 ? faker.random.words() : null),
   type: (i) => types[i % types.length],
-  /**
-   * Generates attributes fields by type.
-   */
-  afterCreate(target) {
-    target.update({
-      attributes: {
-        default_port: faker.datatype.number(),
-      },
-    });
+  default_port() {
+    return this.type === TYPE_TARGET_TCP ? 443 : 22;
```

Comment:
TCP is the default type, but this reads as if, implicitly, SSH is the default.  While not critical right now, this could also be a `switch` or use a data structure to make it easier to add additional type->port mappings.
```suggestion
    return this.type === TYPE_TARGET_SSH ? 22 : 443;
```

Reply:
Since I am rolling back the `id` changes based on type, would you like me to roll this back as well? We aren't explicitly using it anywhere.

Reply:
Rolled back.